bridges/ContainerLinuxReleasesBridge.php

```diff
@@ -40,18 +40,29 @@ public function collectData() {
 
 			$item['uri'] = "https://coreos.com/releases/#$releaseVersion";
 			$item['title'] = $releaseVersion;
-			$item['content'] = nl2br($release['release_notes']);
 
-			$item['content'] .= <<<EOT
-<br/>
+			$content = $release['release_notes'];
+			$content .= <<<EOT
+
 Major Software:
-<br/>
-- Kernel: {$release['major_software']['kernel'][0]}<br/>
-- Docker: {$release['major_software']['docker'][0]}<br/>
-- etcd: {$release['major_software']['etcd'][0]}<br/>
+* Kernel: <tt>{$release['major_software']['kernel'][0]}</tt>
+* Docker: <tt>{$release['major_software']['docker'][0]}</tt>
+* etcd: <tt>{$release['major_software']['etcd'][0]}</tt>
 EOT;
 			$item['timestamp'] = strtotime($release['release_date']);
 
+			// Based on https://gist.github.com/jbroadway/2836900
+			$regex = '/\[([^\[]+)\]\(([^\)]+)\)/';
+			$replacement = '<a href=\'\2\'>\1</a>';
+
+			$item['content'] = preg_replace($regex, $replacement, nl2br($content));
+
+			$regex = '/\n[\*|\-](.*)/';
+			$item['content'] = preg_replace_callback ($regex, function($regs) {
+				$item = $regs[1];
+				return sprintf ("\n<ul>\n\t<li>%s</li>\n</ul>", trim ($item));
```

Comment:
I don't think adding newlines and tabs in HTML is what you intended here (unless you want to format the source code). Actually they are not required for the HTML to be displayed correctly. Try this alternative solution (also adds the same styling to the list as before):

`"<ul style=\"padding-left: 1em;\"><li>%s</li></ul>"`

Reply:
Sometimes I forget we do this to generate RSS feeds, not HTML pages :man_facepalming: 

You are right, formatting is generally responsibility of the feed reader. I'm not sure if global styling in HTML view is a good idea, as it would affect all bridges. Maybe we just keep it simple as is, hoping for the feed readers to take care of it.